src/ert/_c_wrappers/enkf/_deprecation_migration_suggester.py

```diff
@@ -31,7 +31,8 @@ def __init__(self, parser: "ConfigParser", pre_defines: Dict[str, str]):
         "LOG_LEVEL",
         "ENKF_RERUN",
     ]
-    RSH_KEYWORDS = ["RSH_HOST", "RHS_COMMAND"]
+    RSH_KEYWORDS = ["RSH_HOST", "RHS_COMMAND", "MAX_RUNNING_RSH"]
```

Comment:
I know this is not your change, but i suspect the `RHS_COMMAND` should have been `RSH_COMMAND` ?

Reply:
I'll fix that

Reply:
:+1: